code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -5092,86 +5099,138 @@
 
 ////////////////SLIDERS////////////////
 
+/obj/item/weapon/reagent_containers/food/snacks/multispawner/slider
+	name = "sliders"
+	child_type = /obj/item/weapon/reagent_containers/food/snacks/slider
+	child_volume = 2.5
+
+/obj/item/weapon/reagent_containers/food/snacks/multispawner/slider/New()
+	..()
+	reagents.add_reagent(NUTRIMENT, 10) //spawns 4
+
 /obj/item/weapon/reagent_containers/food/snacks/slider
 	name = "slider"
 	desc = "It's so tiny!"
 	icon_state = "slider"
 	food_flags = FOOD_MEAT
+	bitesize = 1.5
+
+/obj/item/weapon/reagent_containers/food/snacks/multispawner/slider/synth
+	name = "synth sliders"
+	child_type = /obj/item/weapon/reagent_containers/food/snacks/slider/synth
 
-/obj/item/weapon/reagent_containers/food/snacks/slider/New()
+/obj/item/weapon/reagent_containers/food/snacks/multispawner/slider/synth/New()
 	..()
-	reagents.add_reagent(NUTRIMENT, 2.5)
-	bitesize = 1.5
+	reagents.add_reagent(NUTRIMENT, 10) //spawns 4
```

Comment:
wouldn't this make them have 20 nutrient, since you're calling the parent multispawner which adds 10
then adding another 10 here

Reply:
The parent new() gets overwritten so no.

Reply:
Nevermind, tested it. It does inherit. Fixing.